testing.md

```diff
@@ -3,22 +3,28 @@ title: Testing
 ---
 
 So you want to easily test the code you're writing? The following
-recipes cover how to write automated tests and see their code
+recipe covers how to write automated tests and see their code
 coverage. We also give pointers to plug those in modern continuous
-integration services like Travis CI and Coveralls.
+integration services like GitHub Actions, Gitlab CI, Travis CI or Coveralls.
 
-[Prove](https://github.com/fukamachi/prove) used to be a widely liked testing framework but its repository was later archived. Its successor [Rove](https://github.com/fukamachi/rove) is not stable enough, so we will be using a mature testing framework called [FiveAM](https://github.com/lispci/fiveam). There are also some [other testing frameworks](https://github.com/CodyReichert/awesome-cl#unit-testing) to explore.
+We will be using a mature testing framework called
+[FiveAM](https://github.com/lispci/fiveam). It supports test suites,
+random testing, test fixtures (to a certain extent) and, of course,
+interactive development.
 
-FiveAM only has an [API doc](https://common-lisp.net/project/fiveam/docs/index.html) as official documentation. You may inspect it or simply read the docstrings in code. Most of the time, they would provide sufficient information that answers your questions. 
+Previously on the Cookbook, the recipe was cooked with [Prove](https://github.com/fukamachi/prove). It used to be a widely liked testing framework but, because of some shortcomings, its repository was later archived. Its successor [Rove](https://github.com/fukamachi/rove) is not stable enough and lacks some features, so we didn't pick it. There are also some [other testing frameworks](https://github.com/CodyReichert/awesome-cl#unit-testing) to explore if you feel like it.
+
+FiveAM has an [API documentation](https://common-lisp.net/project/fiveam/docs/index.html). You may inspect it or simply read the docstrings in code. Most of the time, they would provide sufficient information that answers your questions… if you didn't find them here. Let's get started.
 
 ## Testing with FiveAM
 
 FiveAM has 3 levels of abstraction: check, test and suite. As you may have guessed:
-1. A check is a single assertion that asserts a boolean value. 
-2. A test is the smallest runnable unit. A test case that may contain multiple checks. Any check failure leads to failure of the whole test.
-3. A suite is a collection of tests. When a suite is run, all tests inside would be performed. Suite allows paternity, which means that running a suite will run all the tests defined in it and in its children suites.
 
-A simplest code sample containing 3 basic blocks mentioned above can shown as below:
+1. A **check** is a single assertion that asserts a boolean value. The most used check is `is`, followed by an assertion of yours. For example, `(is (= 2 (+ 1 1)))`.
```

Comment:
Too many times "assertion". Is it really a **check** in FiveAM? Or can we call it simply an assertion?

```suggestion
1. A **check** or assertion. The most used check is `is`, which asserts that its single argument is truthy. For example, `(is (= 2 (+ 1 1)))`.
```

Reply:
yes it is called a check in fiveam.

Reply:
"1. A **check** is a single assertion that checks that its argument is truthy. The most used check is `is`. For example, `(is (= 2 (+ 1 1)))`."

Reply:
Looks great ;)